php7/memcache_pool.c

```diff
@@ -44,7 +44,6 @@ ZEND_DECLARE_MODULE_GLOBALS(memcache)
 MMC_POOL_INLINE void mmc_buffer_alloc(mmc_buffer_t *buffer, unsigned int size)  /*
 	ensures space for an additional size bytes {{{ */
 {
-	register size_t newlen;
 	smart_string_alloc((&(buffer->value)), size, 0);
```

Comment:
@remicollet This line has to be there. Without it I am getting this in PHP 7.0 (Windows, x86, nts):
```
ext\memcache\php7\memcache_pool.c(47): error C2065: 'newlen': undeclared identifier
ext\memcache\php7\memcache_pool.c(47): warning C4018: '>=': signed/unsigned mismatch
```

Reply:
very strange, I indeed see a newlen in smart_string_alloc macro definition, but build passes with 7.0 on linux... one more strange thing...

BTW, re-added.